### PR TITLE
DB-10336 NSDS Spark 3.0 Add Missing Logic

### PR DESCRIPTION
**Pull request by jpanko1**
NSDS v1 updated.
NSDS v2 already had the right logic.

### CODE REVIEW

**Comment by jpanko1** [comment]
jenkins please test branch @cdh6.3.0,skipTestsLongerThan2Minutes

**Comment by spliceci** [comment]
TEST SUCCEEDED +1

**Review by msirek — APPROVED** [review]

**Review by jyuanca — APPROVED** [review]

**Review by carolp-503 — APPROVED** [review]
+1

File: splice_spark/src/main/spark3.0/com/splicemachine/spark/splicemachine/SplicemachineContext.scala [hunk 1]
```diff
@@ -89,7 +104,14 @@ class SplicemachineContext(options: Map[String, String]) extends Serializable {
     val dbProperties = new Properties
     dbProperties.put("useSpark", "true")
     dbProperties.put(EmbedConnection.INTERNAL_CONNECTION, "true")
-    maker.createNew(dbProperties)
+    maker.createNew(
+      if( url.contains("/") ) {  // url = jdbc:splice://localhost:1527/splicedb;user=userid;password=pwd
```
[the remark this answers is not in the document]
Comment (jpanko1) [reply splice_spark/src/main/spark3.0/com/splicemachine/spark/splicemachine/SplicemachineContext.scala:108]: The logic actually allows any DB name, password, or other params that the user passes in.  The comments are meant as an example to show how the parts of the URL get parsed.

**Pull request merged**